src/NosCore.GameObject/Character.cs

```diff
@@ -1370,40 +1373,40 @@ public EqPacket GenerateEq()
             };
         }
 
-        public void RemoveSp()
+        public async Task RemoveSp()
         {
             UseSp = false;
             Morph = 0;
             MorphUpgrade = 0;
             MorphDesign = 0;
             LoadSpeed();
-            SendPacket(this.GenerateCond());
-            SendPacket(GenerateLev());
+            await SendPacket(this.GenerateCond());
+            await SendPacket(GenerateLev());
             SpCooldown = 30;
-            SendPacket(this.GenerateSay(
+            await SendPacket(this.GenerateSay(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)